pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -93,42 +94,43 @@ func parseWaitBehavior(wait string, details *jobspb.ScheduleDetails) error {
 	return nil
 }
 
-func setScheduleOptions(
-	evalCtx *tree.EvalContext, opts map[string]string, sj *jobs.ScheduledJob,
-) error {
-	if v, ok := opts[optFirstRun]; ok {
-		firstRun, _, err := tree.ParseDTimestampTZ(evalCtx, v, time.Microsecond)
-		if err != nil {
-			return err
-		}
-		sj.SetNextRun(firstRun.Time)
-	}
-
+func makeScheduleDetails(opts map[string]string) (jobspb.ScheduleDetails, error) {
 	var details jobspb.ScheduleDetails
 	if v, ok := opts[optOnExecFailure]; ok {
 		if err := parseOnError(v, &details); err != nil {
-			return err
+			return details, err
 		}
 	}
+
 	if v, ok := opts[optOnPreviousRunning]; ok {
 		if err := parseWaitBehavior(v, &details); err != nil {
-			return err
+			return details, err
 		}
 	}
+	return details, nil
+}
 
-	var defaultDetails jobspb.ScheduleDetails
-	if details != defaultDetails {
-		sj.SetScheduleDetails(details)
-	}
+// A sentinel value indicating the schedule should execute based on its
+// schedule default.
+var defaultFirstRun *time.Time
```

Comment:
2¢ but I almost prefer just using `nil` over a _pointer_ sentinel since a) it's obvious it is lack of value, not some "default" value b) no chance someone mutates this _package var_ and creates a really spooky bug.

Reply:
done.